001-quarkus-getting-started/src/test/java/io/quarkus/qe/core/defaultprofile/BaseQuarkusBundleIT.java

```diff
@@ -1,9 +1,9 @@
-package io.quarkus.qe.core;
+package io.quarkus.qe.core.defaultprofile;
 
 import io.quarkus.test.junit.NativeImageTest;
 
 @NativeImageTest
-public class NativeBaseQuarkusBundleIT extends BaseQuarkusBundleTest {
+public class BaseQuarkusBundleIT extends BaseQuarkusBundleTest {
```

Comment:
Why the rename?

Reply:
I just wanted to align the name with other test cases (also in other modules), which use namig convention `XxxTest` + `XxxIT`.

Reply:
We have 41 ***IT.java, 24 of them contain `Native`, also when you look at bootstrapped Quarkus project it has `Native**IT.java` pattern

Reply:
Ok, I didn't inspect the classes properly. `Native` is the way to go, then.